public/locales/zh_Hans/translation.json

```diff
@@ -55,21 +55,7 @@
             "title": "选择存档的目录"
         },
         "wineprefix": "选择WINEPREFIX的文件夹",
-        "yes": "是",
-        "mangohud": "启用MangoHud（需要已安装Mangohud）",
```

Comment:
it seems to some translations have been removed for some reason here. Did you start your branch from the latest main?

Reply:
I _think_ these are redundant translations maybe not removed in a past commit?

![image](https://user-images.githubusercontent.com/30952508/124760054-425cdc80-df28-11eb-8e20-78ee9bfc09e5.png)

There is still a translation string for these strings as you can see in the pic.

I can't find a use of the "box.mangohud" translation key for example, "setting.mangohud" is used however